README.md

```diff
@@ -285,6 +285,24 @@ require 'routemaster/drain/caching_busting'
 $app = Routemaster::Drain::CachingBusting.new
 ```
 
+### Bust Cache for a URL
+
+If a service updates its `_links` one needs to clear caches to force URL
+discovery. Run this (e.g. as a one-off in Hopper):
+
+```ruby
+url = ENV['MY_SERVICE_URL']
+cache = Routemaster::Cache.new
+cache.invalidate(url)
+cache.bust(url)
+```
+
+Note that the URL has to match exactly. For example, `/` suffix would need to be busted separately.
```

Comment:
Does this mean we'd have to do both

/rider_applications/create_fountain_application

AND

rider_applications/create_fountain_application

?

Reply:
No, it means you might need `https://rider-domain.deliveroo.net/` as well as `https://rider-domain.deliveroo.net` depending on what your code does. Unlikely though.